dlc-wallet/src/main/scala/org/bitcoins/dlc/wallet/DLCWallet.scala

```diff
@@ -301,7 +301,7 @@ abstract class DLCWallet
     for {
       feeRate <- feeRateF
       groupedAnnouncements <- groupedAnnouncementsF
-      announcementDataDbs <- announcementDAO.createAll(
+      announcementDataDbs <- announcementDAO.upsertAll(
```

Comment:
can you add a unit test like this to make sure we don't regress in the future? 

https://github.com/bitcoin-s/bitcoin-s/pull/3140/commits/0da17af6f368a678826a4b96841f8e458574d20e#diff-880bebd9e81351c77729c8d406bb5f0fa6b05759c7dccfaec2e3ea845683dbecR360

Reply:
done